diff --git a/nodes.py b/nodes.py
index efdaad3..1536fbd 100644
--- a/nodes.py
+++ b/nodes.py
@@ -7,6 +7,9 @@ COMPARE_FUNCTIONS = {
     "a > b": lambda a, b: a > b,
     "a <= b": lambda a, b: a <= b,
     "a >= b": lambda a, b: a >= b,
+    "pixelwise(a == b)": lambda a, b: (a==b).all(),
+    "shape(a == b)": lambda a, b: a.shape[0] == b.shape[0],
+    "10%": lambda a, b: a.sum()/len(a) > 0.1,
 }
 
 
@@ -233,26 +236,26 @@ class DebugPrint:
 # A dictionary that contains all nodes you want to export with their names
 # NOTE: names should be globally unique
 NODE_CLASS_MAPPINGS = {
-    "Compare-🔬": Compare,
-    "Int-🔬": Int,
-    "Float-🔬": Float,
-    "Bool-🔬": Bool,
-    "String-🔬": String,
-    "If ANY return A else B-🔬": IfExecute,
-    "DebugPrint-🔬": DebugPrint,
-    # "If ANY execute A else B-🔬": IfExecuteNode,
+    "Compare": Compare,
+    "Int": Int,
+    "Float": Float,
+    "Bool": Bool,
+    "String": String,
+    "If ANY return A else B": IfExecute,
+    "DebugPrint": DebugPrint,
+    "If ANY execute A else B": IfExecuteNode,
 }
 
 # A dictionary that contains the friendly/humanly readable titles for the nodes
 NODE_DISPLAY_NAME_MAPPINGS = {
-    "Compare-🔬": "Compare",
-    "Int-🔬": "Int",
-    "Float-🔬": "Float",
-    "Bool-🔬": "Bool",
-    "String-🔬": "String",
-    "If ANY return A else B-🔬": "If ANY return A else B",
-    "DebugPrint-🔬": "DebugPrint",
-    # "If ANY execute A else B-🔬": "If ANY execute A else B",
+    "Compare": "Compare",
+    "Int": "Int",
+    "Float": "Float",
+    "Bool": "Bool",
+    "String": "String",
+    "If ANY return A else B": "If ANY return A else B",
+    "DebugPrint": "DebugPrint",
+    "If ANY execute A else B": "If ANY execute A else B",
 }
 
 
